.gitignore

```diff
@@ -18,7 +18,18 @@ src/tool/users_balance.csv
 src/tool/record_transactions_*.csv
 
 *.DS_Store
-configs
+charts/configs/*
+!charts/configs/lnd
+charts/configs/lnd/*
+!charts/configs/lnd/regtest.yaml
+!charts/configs/lnd/regtest-outside.yaml
+!charts/configs/lnd/localdev.yaml
+!charts/configs/lnd/localdev-outside.yaml
+
+!charts/configs/bitcoind
+charts/configs/bitcoind/*
+!charts/configs/bitcoind/regtest.yaml
+!charts/configs/bitcoind/localdev.yaml
```

Comment:
I feel those config file could be in a different folder but maybe this is for another PR

Reply:
those files are also duplicated in the config repo. probably should be deleted there to avoid confusion.

Reply:
I was planning on keeping a set in configs as well to allow us to overwrite these files without having to commit them to the public galoy repo. On second thought maybe it's something we wouldn't require, so I'll remove it.